tests/integration/validations/factory-general-test.js

```diff
@@ -1040,7 +1040,8 @@ module('Integration | Validations | Factory - General', function(hooks) {
     assert.equal(child.get('validations.errors.length'), 1);
   });
 
-  test('call super in validations class with no super property', function(assert) {
+  // https://github.com/offirgolan/ember-cp-validations/pull/656
+  skip('call super in validations class with no super property', function(assert) {
```

Comment:
do we not want to address this before moving forward here?

Reply:
Decision from @offirgolan was to [skip for now](https://github.com/offirgolan/ember-cp-validations/pull/662#issuecomment-534187453). At this point getting this updated for Ember 3.13 is getting past critical as it's a blocker for many apps to provide feedback on the Octane beta.

Reply:
okay... I would just be worried about apps not working correctly if we skip this test. 🤔

Reply:
Me too, I don't actually know what any of this stuff does! I'm just playing referee to get the approved path merged. I'm happy to investigate and do more, but getting this merged has been a *very* long process so far. Makes me very hesitant to add any more complexity that could delay it further.

Reply:
@Turbo87 skipping as this is already broken on the latest beta. Will have to circle back and figure out a good solution for this.